src/app/features/home/capture-tab/capture-tab.component.ts

```diff
@@ -39,4 +33,17 @@ export class CaptureTabComponent {
   ): number {
     return a.key > b.key ? -1 : b.key > a.key ? 1 : 0;
   }
+
+  // tslint:disable-next-line: prefer-function-over-method
+  trackCaptureGroupByDate(
+    _: number,
+    item: { key: string; value: CaptureItem[] }
+  ) {
+    return item.key;
+  }
+
+  // tslint:disable-next-line: prefer-function-over-method
+  trackCaptureItem(_: number, item: CaptureItem) {
+    return `${item.oldProofHash}_${item.proof?.willCollectTruth}`;
```

Comment:
It seems track by both `oldProofHash` and `willCollectTruth` makes Angular to re-render the CaptureItem when truth is collected, causing obvious flickering. I'm not sure about the necessity of combining them as tracking index here.